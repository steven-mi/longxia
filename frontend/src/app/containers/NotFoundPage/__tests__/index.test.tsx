import * as React from 'react';
import { render } from '@testing-library/react';
import { HelmetProvider } from 'react-helmet-async';

import { NotFoundPage } from '..';

const renderComponent = () =>
  render(
    <HelmetProvider>
      <NotFoundPage />
    </HelmetProvider>,
  );

describe('<NotFoundPage />', () => {
  it('should match the snapshot', () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
