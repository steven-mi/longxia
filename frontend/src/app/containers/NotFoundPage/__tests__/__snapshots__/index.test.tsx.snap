// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match the snapshot 1`] = `
.c2 {
  font-size: 1rem;
  line-height: 1.5;
  color: black;
  margin: 0.625rem 0 1.5rem 0;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 320px;
}

.c1 {
  margin-top: -8vh;
  font-weight: bold;
  color: black;
  font-size: 3.375rem;
}

.c1 span {
  font-size: 3.125rem;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      4
      <span
        aria-label="Crying Face"
        role="img"
      >
        ðŸ˜¢
      </span>
      4
    </div>
    <p
      class="c2"
    >
      404
    </p>
  </div>
</div>
`;
